README.md

```diff
@@ -99,53 +99,27 @@ After PostgreSQL started up, it's time to adjust some configuration for the quay
 
 ```
 ##### Start psql session in quay-database container
-% podman exec -it quay-database /usr/bin/psql
+% podman exec -it quay-database /usr/bin/psql -d quaydb
 psql (10.15)
 Type "help" for help.
 
-##### Create quay database
-postgres=# CREATE DATABASE quaydb;
-CREATE DATABASE
-
-##### Switch to quay database
-postgresql=# \c quaydb
-
 ##### Create pg_trgm extension for quay database
 quaydb=# CREATE EXTENSION IF NOT EXISTS pg_trgm;
 CREATE EXTENSION
-
-##### Create quay database user and grant permissions
-quaydb=# CREATE USER quayuser WITH PASSWORD 'quaypass' CREATEDB;
-CREATE USER
-quaydb=# GRANT ALL PRIVILEGES ON DATABASE quaydb to quayuser;
-GRANT
 <ctrl+d>
 ```
 
 ## Configure Clair database
 
 ```
 ##### Start psql session in quay-database container
-% podman exec -it quay-clair-database /usr/bin/psql
+% podman exec -it quay-clair-database /usr/bin/psql -d clair
```

Comment:
Same here - so `podman exec -it quay-clair-database /usr/bin/psql -d clair -U postgres` should be used.

Reply:
added -U postgres